src/Strategies/RegexLoader/Loaders/WildcardRegex.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace YorCreative\Scrubber\Strategies\RegexLoader\Loaders;
+
+use HaydenPierce\ClassFinder\ClassFinder;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Config;
+
+class WildcardRegex extends NamespaceLoader
+{
+
+    public function canLoad(): bool
+    {
+        return in_array('*', Config::get('scrubber.regex_loader'));
+    }
+
+    public function load(Collection &$regexCollection): void
+    {
+        foreach ($this->getNamespaces() as $namespace) {
+
+            foreach ($this->getRegexClassesInNamespace($namespace) as $fqcn) {
+                $this->loadRegex($fqcn, $regexCollection);
```

Comment:
I am not a fan of the `$fqcn` and would rather have `$fullyQualifiedClassName` instead for better readability. Since it's a preference, I'll let it slide for the 3 regex classes 👍🏼